configs/prod/nfts.json

```
@@ -22,5 +22,8 @@
     },
     {
         "contract": "0xBd3531dA5CF5857e7CfAA92426877b022e612cf8"
+    },
+    {
+        "contract": "0xbc4ca0eda7647a8ab7c2061c2e118a18a936f13d"
```

Addresses should always be checksummed
```suggestion
        "contract": "0xBC4CA0EdA7647A8aB7C2061c2E118A18a936f13D"
```